torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -23,10 +24,13 @@
 #include "torch_xla/csrc/version.h"
 
 // [Implementation Guidelines]
-// - If you want to call a at::func which doesn't exist in AtenXlaType,
-//   call at::native::func instead.
-//   E.g. don't call tensor.is_floating_point() or
-//   at::is_floating_point(tensor), use at::native::is_floating_point(tensor).
+// - If you want to call a at::func which doesn't have a kernel registered
+// according to xla_native_functions.yaml,
+//   you can call a boxed CPU fallback kernel instead.
+//   E.g. don't call tensor.op() or at::op(tensor).
+//   use at::native::call_fallback_fn<&xla_cpu_fallback,
+//         ATEN_OP2(op_name, overload_name)>::call(args...)
```

Comment:
Could you add the comment to cover the difference between `ATEN_OP` and `ATEN_OP2`?

Reply:
will do! I'll also add a link to the pytorch-side documentation